src/scitokens_internal.h

```diff
@@ -111,39 +112,55 @@ friend class scitokens::Validator;
 
 public:
     SciToken(SciTokenKey &signing_algorithm)
-        : m_builder(jwt::create()),
-          m_key(signing_algorithm)
+        : m_key(signing_algorithm)
     {}
 
     void
     set_claim(const std::string &key, const jwt::claim &value) {
-        m_builder.set_payload_claim(key, value);
+        m_claims[key] = value;
         if (key == "iss") {m_issuer_set = true;}
     }
 
+    const jwt::claim
+    get_claim(const std::string &key) {
+        return m_claims[key];
+    }
+
+    const std::string
+    get_claim_string(const std::string &key) {
+        return m_claims[key].as_string();
```

Comment:
This can throw an exception if the claim is not a string, right?  Let's note it in the function comments.